### DISCUSSION TITLE
Elevated Entry/Exit Underpass Needs Chain Signal

### DISCUSSION THREAD

**Discussion by crackalackintv** [Opinionated-Blueprints/10-Books-Full-of-Rails#56] [Ideas]
With the current iteration of the Elevated Entry/Exit Underpass, two trains can get deadlocked facing each other in the "exit" double-signal. Given this is intended to be a single-tracked exit, it needs a change in signaling as shown below.

Also, does putting down this blueprint over existing elevated rails break the straight rails and leaves gaps for anyone else as well? It is, pretty sure, a bot order issue, as it doesn't happen in the editor. 🤔 

![20241119144526_1](https://github.com/user-attachments/assets/70317cdd-de9d-4052-aab0-b14ff168c513)
![20241119144655_1](https://github.com/user-attachments/assets/4fe5afda-cb28-4024-a12b-a5e0b45a5444)

**Comment by Bocian-1** [comment]
This should be an issue

**Reply by crackalackintv** [reply]
Wasn't sure if I should first discuss it here. Created #62.
